src/models/Settings.js

```diff
@@ -62,10 +62,18 @@ export default Model.extend({
     // IDX API VERSION
     apiVersion: ['string', true, '1.0.0'],
 
+    // <OIE>
     // attribute to hold proxy (fake) idx response
     // to render static pages without initiating idx pipeline
     proxyIdxResponse: ['object', false],
 
+    // <OIE>
+    // By default, state handle will be saved to session storage
+    // and will be clear when terminal error or success redirect.
+    // Set this flag to true if you want to override this behavior.
+    // a.k.a dishonor the state handle stored in session storage.
+    overrideExistingToken: ['boolean', false, false],
```

Comment:
nit: Since 'token' is generic, can we rename this to: `overrideExistingStateHandle` or `overrideExistingStateToken`?

Reply:
doesn't have to same. `overrideExistingStateToken` makes more sense for widget.